frontend/src/scenes/pipeline/pipelineLogic.tsx

```diff
@@ -1,6 +1,67 @@
-import { kea } from 'kea'
+import { actions, kea, path, reducers, selectors } from 'kea'
 import type { pipelineLogicType } from './pipelineLogicType'
+import { actionToUrl, urlToAction } from 'kea-router'
+import { urls } from 'scenes/urls'
+import { Breadcrumb, PipelineTabs } from '~/types'
 
-export const pipelineLogic = kea<pipelineLogicType>({
-    path: ['scenes', 'pipeline', 'pipelineLogic'],
-})
+export const singularName = (tab: PipelineTabs): string => {
+    switch (tab) {
+        case PipelineTabs.Filtering:
+            return 'filter'
+        case PipelineTabs.Transformations:
+            return 'transformation'
+        case PipelineTabs.Destinations:
+            return 'destination'
```

Comment:
nit: Maybe we should make these all verbs or nouns:
Option 1 (nouns): filters, transformations, destinations
Option 2 (verbs): filter, transform, export?